src/Opulence/Routing/Dispatchers/RouteDispatcher.php

```diff
@@ -73,62 +71,54 @@ public function dispatch(CompiledRoute $route, Request $request, &$controller =
      * @param CompiledRoute $route The route being dispatched
      * @return Response Returns the value from the controller method
      * @throws RouteException Thrown if the method could not be called on the controller
-     * @throws HttpException Thrown if the controller threw an HttpException
      */
     private function callController($controller, CompiledRoute $route) : Response
     {
-        try {
-            if (is_callable($controller)) {
+        if (is_callable($controller)) {
```

Comment:
Is there any reason to not keep the original `try/catch` and just change `catch (Exception $ex)` to `catch (ReflectionException $ex)`?  Would that accomplish the same thing?

Reply:
Yes actually that may be even cleaner, but it will catch all Reflection errors not depending on where they happened. This may be fine, but I think the more granular check is better here (as this wraps any user land code that may have an error).

What do you think?

Reply:
Ah, you're right.